Dockerfile.openshift.rhel7

```diff
@@ -0,0 +1,12 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.17 AS builder
+WORKDIR /go/src/github.com/openshift/azure-disk-csi-driver
+COPY . .
+RUN make azuredisk ARCH=$(go env GOARCH) && cp _output/$(go env GOARCH)/azurediskplugin .
+
+FROM registry.ci.openshift.org/ocp/4.17:base-rhel9
+COPY --from=builder /go/src/github.com/openshift/azure-disk-csi-driver/azurediskplugin /bin/azurediskplugin
+RUN yum install -y util-linux e2fsprogs xfsprogs ca-certificates && yum clean all && rm -rf /var/cache/yum
+
+LABEL description="Azure Disk CSI Driver"
+
+ENTRYPOINT ["/bin/azurediskplugin"]
```

Comment:
Please add a newline to the end of the file here (the file in master is missing it too, so let's fix it here).